test/parallel/test-compression-decompression-stream.js

```diff
@@ -0,0 +1,41 @@
+// Flags: --no-warnings --expose-internals
+'use strict';
+
+require('../common');
+
+const assert = require('assert');
```

Comment:
nit: for consistency with `node:test`
```suggestion
const assert = require('node:assert');
```